packit_service/service/api/srpm_builds.py

```diff
@@ -0,0 +1,47 @@
+from http import HTTPStatus
```

Comment:
please add the license thingy up here ^

Reply:
Okay

Reply:
hm, we don't have the shortened version Hunor was working on here? nvm